src/main/scala/edu/berkeley/ce/rockslicing/SeedJointSelector.scala

```diff
@@ -0,0 +1,243 @@
+package edu.berkeley.ce.rockslicing
+
+import scala.annotation.tailrec
+
+/**
+  * Selects joints from the input joint set that will be used to seed the RDD in order to maintain
+  * approximately balanced loads among processors.
+  */
+object SeedJointSelector {
+
+  /**
+    * Searches though sequence of joint sets to find best seed joints
+    *
+    * @param jointSets Seq of Seq's of Joints representing input joint sets
+    * @param inputVolume Block representing input rock volume
+    * @param numProcessors Number of processors to be used in the analysis
+    * @return Returns a Seq containing the seed joints that best divide the input volume. If unable to find
+    *         satisfactory seed joints, returns None.
+    */
+  @tailrec
+  def searchJointSets(jointSets: Seq[Seq[Joint]], inputVolume: Block,
+                      numProcessors: Int): Option[Seq[Joint]] = {
+    if (jointSets.isEmpty) {
+      println("ERROR: Unable to find seed joints")
+      None
+    } else if (jointSets.head(0).shape.isEmpty) {
+      // Check that joint set is persistent
+      val seedJointOption = findSeedJoints(jointSets.head, inputVolume, numProcessors, inputVolume.volume)
+
+      if (seedJointOption.isEmpty) {
+          searchJointSets(jointSets.tail, inputVolume, numProcessors)
+      } else {
+        seedJointOption
+      }
+    } else {
+      searchJointSets(jointSets.tail, inputVolume, numProcessors)
+    }
+  }
+
+  /**
+    * Cycles through persistent joint set to find joints that approximately balances load across processes
+    *
+    * @param jointSet Seq of Joints representing a persistent joint set
+    * @param rockVolume Block representing input rock volume that should be divided into approximately equal pieces
+    *                   for each process to operate on
+    * @param numProcessors Number of processes that will be used in analysis
+    * @param totalVolume Total volume of rockVolume
```

Comment:
Yeah, we can do that. I just passed it as a parameter so it wouldn't have to recalculate the same number over and over again since the total volume won't change.